tutorials/holohub_operators_external_applications/holohubCLI_wrapper

```diff
@@ -16,23 +16,44 @@
 # limitations under the License.
 
 SCRIPT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+REPO_URL="${CLI_REPO_URL:-https://github.com/nvidia-holoscan/holohub.git}"
```

Comment:
Do we expect to ever host the CLI at another repo?

Reply:
could be set to a fork of this repo for example for debugging